legal-api/devops/vaults.gcp.env

```diff
@@ -90,14 +90,6 @@ MRAS_SVC_URL="op://namex/$APP_ENV/mras-api/MRAS_SVC_URL"
 MRAS_SVC_API_KEY="op://namex/$APP_ENV/mras-api/MRAS_SVC_API_KEY"
 VPC_CONNECTOR="op://CD/$APP_ENV/base/VPC_CONNECTOR"
 
-NATS_SERVERS=""
-NATS_CLUSTER_ID="test-cluster"
-NATS_QUEUE="filing-worker"
-NATS_CLIENT_NAME="entity.filing.filer.worker"
-NATS_FILER_SUBJECT="entity.filing.filer"
-NATS_ENTITY_EVENT_SUBJECT="entity.events"
-NATS_EMAILER_SUBJECT="entity.email"
```

Comment:
think we can remove the [config.py](https://github.com/bcgov/lear/blob/3ec6e07aa208af7aced1d5f50c33a022d04c0ecb/legal-api/src/legal_api/config.py#L220-L227) entries too